test/core/models/ocelot/stanzas/ValueStanzaSpec.scala

```diff
@@ -99,29 +102,47 @@ class ValueStanzaSpec extends BaseSpec {
     )
     .as[JsObject]
 
+  val validValueStanzaWithListJson: JsObject = Json
+    .parse(
+      s"""{
+         |  "type": "${stanzaType}",
+         |  "values": [
+         |    {
+         |      "type": "${listType}",
+         |      "label": "${listLabel}",
+         |      "value": "${listValue}"
+         |    }
+         |  ],
+         |  "next": ["${next}"],
+         |  "stack": ${stack}
+         |}
+    """.stripMargin
+    )
+    .as[JsObject]
+
   "ValueStanza" must {
 
-    "deserialise from json" in {
+    "deserialize from json" in {
```

Comment:
Again, mention "scalar" in description, easier to understand what this test is specific for

Reply:
Done